packages/angular/common/src/utils.ts

```diff
@@ -0,0 +1,59 @@
+// TODO: Is there a way we can grab this from angular-component-lib instead?
```

Comment:
So the main problem here is these functions are originally defined in `src/angular-component-lib` which is treated as a separate module than what we are in currently. Former is in `@ionic/angular` latter is in `@ionic/angular/common`.

I could export these functions from `@ionic/angular`, but then they'd be public API which seems strange. I could also have these files be generated in the common directory, but then that means the autogenerated lazy loaded components will be added to the common directory too.

Given that these functions do not often change, the best approach I have right now is to copy it over. Open to other ideas!